packages/ui/src/app/pages/Settings/Settings.tsx

```diff
@@ -1,43 +1,75 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
 
-import { PageLayout, PageHeaderWrapper } from '@/app/components/PageLayout'
+import { PageHeaderWrapper, PageLayout } from '@/app/components/PageLayout'
 import { NetworkType } from '@/app/config'
 import { LanguageSelect } from '@/common/components/LanguageSelect'
-import { MainPanel } from '@/common/components/page/PageContent'
+import NetworkInfo from '@/common/components/NetworkInfo/NetworkInfo'
+import { MainPanel, RowGapBlock } from '@/common/components/page/PageContent'
 import { PageTitle } from '@/common/components/page/PageTitle'
 import { SimpleSelect } from '@/common/components/selects'
+import { Tabs } from '@/common/components/Tabs'
 import { useNetwork } from '@/common/hooks/useNetwork'
+import { useNetworkEndpoints } from '@/common/hooks/useNetworkEndpoints'
+
+type Tab = 'SETTINGS' | 'LANGUAGE'
 
 export const Settings = () => {
   const options: NetworkType[] = ['local', 'local-mocks', 'olympia-testnet']
   const [network, setNetwork] = useNetwork()
+  const { t } = useTranslation('settings')
+  const [endpoints] = useNetworkEndpoints()
+  const [currentTab, setCurrentTab] = useState<Tab>('SETTINGS')
+  const tabs = [
+    { title: t('network'), active: currentTab === 'SETTINGS', onClick: () => setCurrentTab('SETTINGS') },
+    { title: t('settings'), active: currentTab === 'LANGUAGE', onClick: () => setCurrentTab('LANGUAGE') },
```

Comment:
One last details, I think this second tab is supposed to be called language

Reply:
done 👍